crates/cli/src/components.rs

```diff
@@ -710,7 +711,23 @@ where
 
         executor.spawn_critical("amm quoting service", amm);
 
-        // TODO: Manage runtime here instead of using ConsensusManager
+        let driver = RollupDriver::new(
+            block_height,
+            // TODO(mempirate): Replace with config
+            Duration::from_millis(2000),
```

Comment:
```suggestion
            Duration::from_millis(config.rollup_block_time_ms)),
```

Reply:
Will do in follow-up, we don't have this yet